commcare_connect/templates/tailwind/pages/opportunity_dashboard/opportunity_resource_modal.html

```diff
@@ -0,0 +1,89 @@
+<div x-show="showModal"
+    x-transition:enter="transition ease-out duration-300"
+    x-transition:enter-start="opacity-0"
+    x-transition:enter-end="opacity-100"
+    x-transition:leave="transition ease-in duration-200"
+    x-transition:leave-start="opacity-100"
+    x-transition:leave-end="opacity-0"
+    class="fixed inset-0 z-50 overflow-y-auto"
+    style="display: none;">
+
+    <!-- Background overlay -->
+    <div class="fixed inset-0 bg-black/50 bg-opacity-50 transition-opacity"
+            @click="showModal = false;document.body.style.overflow = '';"></div>
+
+    <!-- Modal content -->
+    <div class="flex items-center justify-center min-h-screen p-4">
+        <div class="relative rounded-2xl bg-gray-100 shadow-xl w-2/3 h-1/2 mx-auto overflow-hidden p-3"
+                @click.away="showModal = false;document.body.style.overflow = '';">
+            <!-- Modal header with tabs -->
+            <div class="flex flex-col w-full" x-init="selectedTab='Learn App'">
+                <!-- Tabs -->
+                <div class="flex relative mx-auto items-center shadow rounded-lg justify-between w-full px-3 mb-3 bg-slate-50 h-14">
+                    <ul class="tabs">
+                        <li
+                            @click="selectedTab = 'Learn App'"
+                            :class="{ 'active': selectedTab === 'Learn App' }">
+                            <span>Learn App</span>
+                            <div x-show="selectedTab === 'Learn App'"></div>
+                        </li>
+                        <!-- Delivery App Tab -->
+                        <li @click="selectedTab = 'Delivery App'"
+                            :class="{ 'active': selectedTab === 'Delivery App' }">
+                            <span>Delivery App</span>
+                            <div x-show="selectedTab === 'Delivery App'"></div>
+                        </li>
+                        <!-- Payment Units Tab -->
+                        <li @click="selectedTab = 'Payments Units'"
+                            :class="{ 'active': selectedTab === 'Payments Units' }">
+                            <span>Payment Units</span>
+                            <div x-show="selectedTab === 'Payments Units'"></div>
+                        </li>
+                    </ul>
+
+                    <!-- Close button -->
+                    <button @click="showModal = false;document.body.style.overflow = '';" id="button" class="button-icon" >
+                        <i class="fa-light fa-xmark-large text-brand-deep-purple"></i>
+                    </button>
+                </div>
+
+                <!-- Modal body / Tab content -->
+                <div class="z-50 bg-white ">
+                    <!-- Learn App Content -->
+                    <div x-show="selectedTab === 'Learn App'"
+                            hx-get="{% url 'opportunity:learn_module_table' request.org.slug opp_id %}"
+                            hx-trigger="tabShown from:body"
```

Comment:
This doesn't need to get triggered each time `tabShown`, only the first time, right? If so, you can use a pattern like this one
https://github.com/dimagi/commcare-connect/pull/583/files#diff-12b291e8c245ca3f054fa30b3c6eedde44b28d40851a868862a6e59069000c10R20

```
     x-init="$watch('isOpen', value => {
         if (value && !historyLoaded) {
             // Only hx-trigger first time
             $dispatch('load-payment-history'); 
             historyLoaded = true; 
         }
     })"
```